app/logic/Mail/EWS/EWSFolder.ts

```diff
@@ -378,7 +378,7 @@ export class EWSFolder extends Folder {
   }
 
   async moveFolderHere(folder: EWSFolder) {
-    assert(folder.account = this.account, "Cannot move folders between accounts");
+    assert(folder.account == this.account, "Cannot move folders between accounts");
```

Comment:
Whoops. Thanks for finding this.

Reply:
Looks like IMAP is also affected, presumably because I originally copied the assert from there.